src/Node/Expression/CallExpression.php

```diff
@@ -293,11 +296,16 @@ private function reflectCallable($callable)
         }
 
         if (\is_array($callable)) {
-            if (!method_exists($callable[0], $callable[1])) {
-                // __call()
+            list($class, $method) = $callable;
```

Comment:
The naming `$class` is confusing here, as this is really `$objectOrClass`

Reply:
Is it ok that I destructed the callable?